e2e/support/helpers/e2e-embedding-iframe-sdk-helpers.ts

```diff
@@ -50,6 +42,17 @@ export interface BaseEmbedTestPageOptions {
   onVisitPage?(): void;
 }
 
+export interface MetabaseElement {
+  // The component to embed
+  component: "metabase-dashboard" | "metabase-question" | "metabase-browser";
```

Comment:
(FYI, it was called an element because it's technically a [HTML5 Custom Element](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_custom_elements) now, but component is also fine.)